css/css-text/i18n/ja/css-text-line-break-ja-in-loose.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html  lang="en" >
+<head>
+<meta charset="utf-8"/>
+<title>CSS text, linebreaks: IN (loose,ja)</title>
+<link rel="author" title="Richard Ishida" href="mailto:ishida@w3.org">
+<link rel="help" href="https://drafts.csswg.org/css-text-3/#line-break-property">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<meta name="assert" content="When the language is Japanese, and line-break:loose, a browser allows a break before an inseparable character.">
+<style type="text/css">
+@font-face {
+    font-family: 'mplus-1p-regular';
+    src: url('/fonts/CSSTest/mplus-1p-regular.woff') format('woff');
+    }
+#wrapper { position: relative; }
+.test { color: red; }
+.test, .ref { font-size: 30px; font-family: mplus-1p-regular, sans-serif; width: 185px; padding: 0; border: 1px solid orange; line-height: 1em; }
+</style>
+<style>
+.test { line-break: loose; }
+</style>
+<script>
+var charlist = `2024  ONE DOT LEADER
+2025  TWO DOT LEADER
+2026  HORIZONTAL ELLIPSIS
+22EF  MIDLINE HORIZONTAL ELLIPSIS
+FE19  PRESENTATION FORM FOR VERTICAL HORIZONTAL ELLIPSIS
+10AF6  MANICHAEAN PUNCTUATION LINE FILLER`
```

Comment:
U+10AF6 is probably not in 'mplus-1p-regular' and that not testable. We could add a font that has a glyph, or use test font such as CSS Orientation test font, but this isn't a CJK character, probably not worth to test?

Appear in all "*-in-*" files.

Reply:
I agree that probably not worth testing Manichaean in this case. @xfq could you remove that line? (except for the `, of course).

Reply:
Done.